app/assets/Helpers/MapContainer/MockMarkers.js

```diff
@@ -0,0 +1,51 @@
+export const MockMarkers = {
+  Chengdu: {
+    completed: false,
+    selected: false,
+    region: 'China',
+    dreamLocation: "Chengdu",
+    formattedAddress: "Chengdu, Sichuan, China"
+  },
+  DenverCompletedNotSelected: {
+    completed: true,
+    selected: false,
+    region: 'Colorado',
+    dreamLocation: "Denver",
+    formattedAddress: "Denver, Colorado"
+  },
+  DenverNotCompletedSelected: {
+    completed: false,
+    selected: true,
+    region: 'Colorado',
+    dreamLocation: "Denver",
+    formattedAddress: "Denver, Colorado"
+  },
+  Sherlock: {
+    completed: false,
+    selected: true,
+    region: 'Marylebone',
+    dreamLocation: "221 Baker Street, London",
+    formattedAddress: "221B Baker Street, Marylebone, London"
+  },
+  Everest: {
+    completed: false,
+    selected: true,
+    region: 'Nepal',
+    dreamLocation: "Everest",
+    formattedAddress: "Mt Everest"
+  },
+  AcadiaNotCompleted: {
+    completed: false,
+    selected: false,
+    region: 'Maine',
+    dreamLocation: "Acadia National Park",
+    formattedAddress: "Acadia National Park"
+  },
+  AcadiaCompleted: {
+    completed: true,
+    selected: false,
+    region: 'Maine',
+    dreamLocation: "Acadia National Park",
+    formattedAddress: "Acadia National Park"
+  }
+}
```

Comment:
I'm digging how all this data is broken out into objects and passed into your helper file.